src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java

```diff
@@ -224,7 +224,7 @@ public double getSymptomDelay() {
     public double getTransAdjustment() {
     	double transAdjustment = 0.0;
     	if(asymptomatic) transAdjustment = this.asymptomaticTransAdjustment;
-    	else if(phase1 && !this.isSymptomatic()) transAdjustment = this.asymptomaticTransAdjustment;
+    	else if(phase1 && !this.isSymptomatic()) transAdjustment = this.symptomaticTransAdjustment; // Slight adjustment to this to set higher levels of pre-symptomatic transmission
     	else if(phase1 && this.isSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
     	else if(phase2 && this.isSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
```

Comment:
Can we combine the three other branches now since it seems to be you get either asymptomaticAdj or symptomaticAdj? For clarity.

Ideally we change this, but perhaps you know that these values are going to change very soon (in which case we can keep it in).